sonic_platform_base/sonic_xcvr/api/public/sff8636.py

```diff
@@ -403,3 +404,46 @@ def get_lpmode(self):
         # Since typically optics come up by default set to high power, in this case,
         # power_override not being set, function will return high power mode.
         return power_set and power_override
+
+    def get_power_class(self):
+        '''
+        Retrieves power class of the module.
+
+        Returns:
+            int: Power class of the module, -1 if it fails
+        '''
+        power_class_str = self.xcvr_eeprom.read(consts.POWER_CLASS_FIELD)
+        power_class = -1
+
+        if power_class_str is None:
+            return power_class
+        if not power_class_str.startswith(self.POWER_CLASS_PREFIX):
+            return power_class
+
+        prefix_len = len(self.POWER_CLASS_PREFIX)
+        power_class = int(power_class_str[prefix_len:prefix_len + 1])
+
+        return power_class
+
+    def set_high_power_class(self, enable):
```

Comment:
@longhuan-cisco let the argument take pclass, enable two arguments. see other comment below